NETCORE/test/Microsoft.ApplicationInsights.AspNetCore.Tests/Microsoft.ApplicationInsights.AspNetCore.Tests.csproj

```diff
@@ -25,12 +25,19 @@
     </PackageReference>
     <PackageReference Include="xunit" Version="2.4.1" />
     <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="1.0.4" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net46' ">
-    <Reference Include="System" />
-    <Reference Include="Microsoft.CSharp" />
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="1.1.8" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.1' ">
```

Comment:
This is correct. By default all ASP.NET Core 2.1 app had `IHostingEnvironment` reference in `startup.cs`. As we are retiring .NET Core 2.1 we need to move from `IHostingEnvironment` to `IWebHostEnvironment`. This change is not simple, it requires modification to `DefaultApplicationInsightsServiceConfigureOptions.cs`. This is out of the scope for this PR.